install/jaxr/bin/sig-test-pkg-list.txt

```diff
@@ -22,6 +22,4 @@
 # parent package's signature is checked.
 ##
 
-javax.xml.registry
```

Comment:
I think that the install/jaxr folder can be deleted, we probably can look for references to that folder, some of which will be from ant scripts, that need to be partially updated to not reference install/jaxr or jax stuff indirectly.  It is hard to search for these, as jaxr matches a few other longer technology names.